islandora_scholar.drush.inc

```diff
@@ -118,3 +145,170 @@ EOQ;
   $sandbox['offset'] += $citation_update;
   $context['finished'] = $sandbox['offset'] / $sandbox['total'];
 }
+
+
+/**
+ * Command callback to update citations with HTML in MODS.
+ */
+function drush_islandora_scholar_fix_doi_html_in_mods() {
+  $action = drush_get_option('action', variable_get('islandora_doi_handle_imported_metadata', FALSE));
+  $audit = drush_get_option('audit', FALSE);
+  $dry_run = drush_get_option('dry-run', FALSE);
+  $modify_my_data = drush_get_option('modify-my-data', FALSE);
+
+  // Require one and only one of --audit, --dry-run, --encode.
+  if (!($audit || $dry_run || $modify_my_data)) {
```

Comment:
Cool. Thank god we had only 3 options here or the combinatory for those `if`s would have gone crazy!!. Maybe next time better to add a single option with 3 values to write less code, but I'm totally ok with how it is now!! @rosiel++